src/renderer/components/data-settings/data-settings.js

```diff
@@ -263,50 +264,59 @@ export default Vue.extend({
       this.updateShowProgressBar(true)
       this.setProgressBarPercentage(0)
       let count = 0
-      for (let i = 1; i < (youtubeSubscriptions.length - 1); i++) {
-        const channelId = youtubeSubscriptions[i].split(',')[0]
-        const subExists = primaryProfile.subscriptions.findIndex((sub) => {
-          return sub.id === channelId
-        })
-        if (subExists === -1) {
-          let channelInfo
-          if (this.backendPreference === 'invidious') { // only needed for thumbnail
-            channelInfo = await this.getChannelInfoInvidious(channelId)
-          } else {
-            channelInfo = await this.getChannelInfoLocal(channelId)
-          }
 
-          if (typeof channelInfo.author !== 'undefined') {
-            const subscription = {
-              id: channelId,
-              name: channelInfo.author,
-              thumbnail: channelInfo.authorThumbnails[1].url
-            }
-            subscriptions.push(subscription)
+      const ytsubs = youtubeSubscriptions.slice(1).map(yt => {
+        const splitCSVRegex = /(?:,|\n|^)("(?:(?:"")*[^"]*)*"|[^",\n]*|(?:\n|$))/g
```

Comment:
splitting on comma was okay for when we just wanted to get the id but it had to be changed as we would likely want to get the name (in the future) which could have commas is in it